R/branch_metrics.R

```diff
@@ -256,7 +256,7 @@ compare_branchm <- function(test_path, branch1, branch2 = "master") {
   target2 <- git2r::repository(file.path("./"))
   # If branch2 is specified, check out to it and obtain commit list
   if (!is.null(branch2)) {
-    original_state2 <- git2r::head(target2)
+    original_state2 <- git2r::repository_head(target2)
```

Comment:
you have these changes in 3 different branches/prs, right? it is better for review to keep them in just one branch/pr so your reviewers (me) only have to review the change once.

Reply:
Sure, I will keep all the changes in **test-pkg** branch. I will also keep only this PR open.